public/index.html

```diff
@@ -13,8 +13,11 @@
 		gtag('js', new Date());
 
 		gtag('config', 'G-30VQJDBL2T');
-
 	</script>
+  	<script src="https://unpkg.com/vue"></script>
+  	<script src="https://unpkg.com/vueperslides"></script>
+  	<link href="https://unpkg.com/vueperslides/dist/vueperslides.css" rel="stylesheet">
```

Comment:
Is this necessary? I didn't see this in Sanaa changes - NPM should take care of dependencies for us.

Reply:
It was either you include it as a script or install it as a dependency but including it as a script didn't really work so I told her to install it as a dependency for now and then if my pr is merged first, to uninstall on her end and rebase.